IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/wolfssl_demo.c

```diff
@@ -0,0 +1,429 @@
+            printf("ERROR wolfSSL_read: %d\n", wolfSSL_get_error(ssl, 0));
+            ret = -1;
+        }
+        else {
+            rcvBuff[ret] = '\0';
+            printf("Received: %s\n\n", rcvBuff);
+            ret = 0;
+        }
+    }
+
+    
+    wolfSSL_shutdown(ssl);
+
+    FreeRTOS_shutdown(socket, FREERTOS_SHUT_RDWR);
+    while(FreeRTOS_recv(socket, rcvBuff, BUFF_SIZE -1, 0) >=0) {
+    	vTaskDelay(250);
+    }
+
+    FreeRTOS_closesocket(socket);
+
+
+    wolfSSL_free(ssl);
+    wolfSSL_CTX_free(ctx);
+    wolfSSL_Cleanup();
+
+    return;
+}
+
+static void Tls_client_demo(void)
+{
+
+	/* setup ciphersuite list to use for TLS handshake */
+
+#if defined(WOLFSSL_RENESAS_TSIP_TLS)
+
+    #ifdef USE_ECC_CERT
+    const char* cipherlist[] = {
+        "ECDHE-ECDSA-AES128-GCM-SHA256",
+        "ECDHE-ECDSA-AES128-SHA256"
+    };
+    const int cipherlist_sz = 2;
+
+    #else
+    const char* cipherlist[] = {
+        "ECDHE-RSA-AES128-GCM-SHA256",
+        "ECDHE-RSA-AES128-SHA256",
+        "AES128-SHA",
+        "AES128-SHA256",
+        "AES256-SHA",
+        "AES256-SHA256"
+    };
+    const int cipherlist_sz = 6;
+
+    #endif
+
+#else
+    const char* cipherlist[] = { NULL };
+    const int cipherlist_sz = 0;
+
+#endif
+
+    int i = 0;
+
+    printf("/*------------------------------------------------*/\n");
+    printf("    TLS_Client demo\n");
+    printf("    - TLS server address:" TLSSERVER_IP " port: %d\n",
+                                                             TLSSERVER_PORT );
+
+#if defined(WOLFSSL_RENESAS_TSIP_TLS) && (WOLFSSL_RENESAS_TSIP_VER >=109)
+    printf("    - with TSIP\n");
+#endif
+    printf("/*------------------------------------------------*/\n");
+
+    /* setup credentials for TLS handshake */
+#if defined(WOLFSSL_RENESAS_TSIP_TLS) && (WOLFSSL_RENESAS_TSIP_VER >=109)
+
+    #if defined(USE_ECC_CERT)
+
+    /* Root CA cert has ECC-P256 public key */
+    tsip_inform_cert_sign((const byte*)ca_ecc_cert_der_sig);
+
+    #else
+    
+    /* Root CA cert has RSA public key */
+    tsip_inform_cert_sign((const byte*)ca_cert_der_sig);
+
+    #endif
+
+    wc_tsip_inform_user_keys_ex(
+            (byte*)&g_key_block_data.encrypted_provisioning_key,
+            (byte*)&g_key_block_data.iv,
+            (byte*)&g_key_block_data.encrypted_user_rsa2048_ne_key,
+            encrypted_user_key_type);
+
+    guser_PKCbInfo.user_key_id = 0;
+
+#endif /* WOLFSSL_RENESAS_TSIP_TLS && (WOLFSSL_RENESAS_TSIP_VER >=109) */
+
+    do {
+        if(cipherlist_sz > 0 ) printf("cipher : %s\n", cipherlist[i]);
+
+        Tls_client_init(cipherlist[i]);
+
+        Tls_client();
+
+        i++;
+    } while (i < cipherlist_sz);
+
+    printf("End of TLS_Client demo.\n");
+}
+
+
+/* Demo entry function called by iot_demo_runner
+ * To run this entry function as an aws_iot_demo, define this as 
+ * DEMO_entryFUNCTION in aws_demo_config.h.
+ */
+void wolfSSL_demo_task(bool 		awsIotMqttMode,
```

Comment:
Please remove hard tabs in this function definition and body.

Reply:
Replaced tabs with white spaces.

Reply:
Hi @TakayukiMatsuo, this function (wolfSSL_demo_task) still looks like it uses hard tabs throughout the function definition and body.  Can you take another edit pass and change hard tabs to spaces here?

Reply:
Oops! Have fixed another copy.